scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -192,21 +195,19 @@ get_utility_path() {
 get_recovery_state_of_local_node() {
     local state=0
 
-    test="$(lvs -o name $LOCAL_MD_VOLUMEGROUP | grep $SNAPSHOT | awk '{ print $1}')"
+    test=$(get_lvs_info "$LOCAL_NODE" "name" "$LOCAL_MD_VOLUMEGROUP" "$SNAPSHOT" "$1")
     if [ "$test" != "$SNAPSHOT" ]; then
-        test="$(lvs -o name $LOCAL_MD_VOLUMEGROUP | grep $SWAP_DEVICE |  awk '{print $1}')"
+        test=$(get_lvs_info "$LOCAL_NODE" "name" "$LOCAL_MD_VOLUMEGROUP" "$SWAP_DEVICE" "$1")
         if [ "$test" = "$SWAP_DEVICE" ]; then
-            MD_SIZE="$(lvs -o lvname,size $LOCAL_MD_VOLUMEGROUP --units K --nosuffix | \
-                        grep $MD_DEVICE | awk '{ print $2 }')"
+            MD_SIZE=$(get_lvs_info "$LOCAL_NODE" "lvname,size" "$LOCAL_MD_VOLUMEGROUP --units K --nosuffix" "$MD_DEVICE" "$2")
 
-            SWAP_SIZE="$(lvs -o lvname,size $LOCAL_MD_VOLUMEGROUP --units K --nosuffix| \
-                        grep $SWAP_DEVICE | awk '{ print  $2 }')"
+            SWAP_SIZE=$(get_lvs_info "$LOCAL_NODE" "lvname,size" "$LOCAL_MD_VOLUMEGROUP --units K --nosuffix" "$SWAP_DEVICE" "$2")
 
             # If the comparison is true then bc return 1 otherwise return 0
             if [ $(bc <<< "$MD_SIZE > $SWAP_SIZE") = 1 ]; then
```

Comment:
Are you sure the command `bc` is installed on system by default?
If not, this will result in error.
If yes, this is OK.

(My local VM does not have "bc" installed)

Reply:
yes it is installed

Reply:
In VM's bc is not installed by default, better to not use it.

Reply:
Added check to ensure that bc is installed on VM. `bc` is required here to do comparison of float type variables, as metadata size and swap size we get here are of float type.